puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/release_test.yaml

```diff
@@ -1,10 +1,10 @@
 # Testing already-built RPMs from Koji and Debian packages in stagingdeb.
 - job:
     name: release_test
+    node: 'el && ipv6'
     project-type: matrix
     logrotate:
-      daysToKeep: -1
-      numToKeep: 16
+      daysToKeep: 8
```

Comment:
Why change this to days? This is used in the release pipeline which you run at most every few weeks. A number feels logical to me.

Reply:
I agree, looking over the jobs we seem to have a number of ways we handle this over these jobs. I think keeping a number is better overall, as it will let anyone who is unfamiliar with the job see what a run should look like. Also making this uniform across all systest jobs would be awesome.

Reply:
I agree that we should have a unified `days-` and `numToKeep` for all jobs in our jenkins instance, but I believe that is outside the scope of this PR.

Reply:
I set this job's `daysToKeep` and `numToKeep` to be the same as what `systest_foreman` was previously because it isn't a job-chain anymore, but rather a build step of the job in question.